src/google_map.js

```diff
@@ -205,6 +207,10 @@ export default class GoogleMap extends Component {
         this._initMap();
       }
     }, 0, this);
+    if (this.props.resetBoundsOnResize) {
+      const that = this;
+      detectElementResize.addResizeListener(mapDom, that._mapDomResizeCallback);
```

Comment:
I think yes, 

also I'll try to find and give you a solution what to do with resize
I've got why 
```
  _mapDomResizeCallback = () => {
    this.resetSizeOnIdle_ = true;
  }
```

is not enough, I think there should be gmap api trigger to force internal map to resize